### DISCUSSION TITLE
Crash when "save as ocad"

### DISCUSSION THREAD

**Discussion by JoshBeech123** [OpenOrienteering/mapper#1985] [General]
I have been having a problem with one particular map when trying to "save as" Ocad. 
No issues with other maps, but the one I need the most 
Any advice would be very much apricated

**Comment by dg0yt** [comment]
We rarely see crashes. So the only advice is providing the (OMAP) file so that we can try to isolate the issue in a debugger. (Ask for a private upload link via e-mail if needed. My e-mail adress should be visible in the GH profile.)
In any case, make sure you don't use an old version of Mapper. You opened a new "discussion". Our "issue" template would ask for the version.

**Reply by JoshBeech123** [reply]
Thank you very much. 
Not hiding you email address anywhere.

**Reply by dg0yt** [reply]
And there is https://www.openorienteering.org/about.html

**Comment by dl3sdo** [comment]
Is there something special with the map (e.g., many templates, large area, many objects etc.)? Would is still happen if you remove parts of the map?

**Comment by JoshBeech123** [comment]
Thank you very much.  File size =106669 kb (so pretty big). Area = 8km2A quite a lot of templates (80x satellite image 4x geo tif)  CheersJosh Beech From: Matthias KühleweinSent: Wednesday, 4 August 2021 18:44To: OpenOrienteering/mapperCc: JoshBeech123; AuthorSubject: Re: [OpenOrienteering/mapper] Crash when "save as ocad" (#1985) Is there something special with the map (e.g., many templates, large area, many objects etc.)? Would is still happen if you remove parts of the map?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. 

--- 
Edit:

Thank you very much.  File size =106669 kb (so pretty big). Area = 8km2A quite a lot of templates (80x satellite image 4x geo tif) 
Cheers Josh Beech 

> Is there something special with the map (e.g., many templates, large area, many objects etc.)? Would is still happen if you remove parts of the map?

**Reply by dg0yt** [reply]
If you want to sent the map for debugging, ignore the templates and just put the OMAP file in a ZIP.

**Reply by JoshBeech123** [reply]
Thank you very much. I emailed you

**Reply by dg0yt** [reply]
> I emailed you

I'm sorry but I am unable to find an e-mail from you.
